src/main/resources/swagger.api/schemas/EcsTlr.yaml

```diff
@@ -23,6 +23,10 @@ EcsTlr:
       description: "Date when the request expires"
       type: string
       format: date-time
+    requestDate:
```

Comment:
this field is required in the request schema of mod-circulation-storage. Should we make it required here as well?